docs/src/main/asciidoc/_developer-guide/Embedded_API.adoc

```diff
@@ -0,0 +1,110 @@
+[[embedded-api]]
+= Embedded API
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:idprefix:
+:idseparator:       -
+
+include::../common_attributes.adoc[]
+
+The embedded API can be used to launch {appservername} within a currently running process.
+
+The embedded server can be reinitialized with a different JBoss Home. However the module directory, `module.path`
+system property, and the modules system packages, `jboss.modules.system.pkgs` system property, are effectively static.
+This means that creating a new embedded server or host controller within the same VM will not allow overriding the
+modules directory or the system packages.
+
+By default the logging subsystem is not used to configure logging for the embedded server. A hint can be used on the
```

Comment:
After the first sentence there should be a sentence explaining what it used.

Reply:
I probably just need reword this whole paragraph.